src/store/reducers/index.js

```diff
@@ -1,7 +1,11 @@
 import { combineReducers } from 'redux';
 import { cohortsReducer } from './cohorts.reducer';
+import { cohortDataReducer } from './singleCohort.reducer';
 
 // import and add separate reducers into combineReducers
-const rootReducer = combineReducers({ allCohorts: cohortsReducer });
+const rootReducer = combineReducers({
+  allCohorts: cohortsReducer,
+  activeCohort: cohortDataReducer
```

Comment:
If we're storing all of the cohorts data in this section of the store, I think the "activeCohort" cohort key name might be a bit confusing. What about something like "cohortData"?

Reply:
makes sense